src/main/resources/static/_shared.yml

```diff
@@ -4833,10 +4833,10 @@ components:
         - person_name
         - person_risks_tier
       x-enum-varnames:
-        - duration
-        - expectedArrival
-        - createdAt
-        - applicationSubmittedAt
+        - DURATION
+        - EXPECTED_ARRIVAL
+        - CREATED_AT
+        - APPLICATION_SUBMITTED_AT
```

Comment:
kind of weird there are only 4 in `x-enum-varnames` but more in `enum`. Any idea what happened here?